BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/distance/BeaconDistanceCalculatorTest.java

```diff
@@ -58,7 +79,7 @@ public void calculateDistance_calibratedRssi_calibratedDistance() throws Excepti
         assertEquals(1, calculatedDistance, 0.1);
     }
 
-    public Beacon createDummyBeacon() {
+    public Beacon createLowAltitudeDummyBeacon() {
```

Comment:
Add parameters for `altitude` and `location` instead of having 2 nearly equal methods.